pspgl/PSPBUILD

```diff
@@ -1,40 +1,40 @@
 pkgname=pspgl
-pkgver=r12
-pkgrel=1
+pkgver=0
+pkgrel=2
 pkgdesc="OpenGL-ESish library for PSP"
 arch=('mips')
-url="https://github.com/pspdev/psp-ports"
+url="https://github.com/pspdev/${pkgname}"
 license=('BSD')
 depends=()
 makedepends=()
 optdepends=()
-source=("git+https://github.com/pspdev/pspgl.git")
+source=("git+https://github.com/pspdev/${pkgname}.git")
 sha256sums=('SKIP')
 
-pkgver() {
-    cd pspgl
+# pkgver() {
```

Comment:
I would just remove this instead of commenting it out, otherwise this is going to be confusing later.